PubSub/build.gradle

```diff
@@ -0,0 +1,27 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:1.5.0'
+
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
```

Comment:
If an app goes to integrate pubsub, it's simple enough compiling the JAR and copying it to the libs directory, but it would also be really convenient to start publishing this and the Lantern SDK to Sonatype's snapshots repository. We don't have to do it in this PR, of course, but I figured its worth mentioning.